### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: "Refresh", "Sort by" buttons and "Category" combo box are not accessible with keyboard in Browse repositories dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1432]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. IntelliJ IDEA start screen appears, Navigate and select  Plugins under Configure combo box.
3. Plugins dialog will open.
4. In plugins dialog navigate to "Browse repositories " button and hit enter.
5. In "Browse repositories " dialog Navigate to "Refresh", "Sort by" buttons and "Category" combo box and try to access with keyboard.

**Expected Result:**
User should be able to access the "Refresh", "Sort by" buttons and "Category" combo box using keyboard.

**Actual Result:**
"Refresh", "Sort by" buttons and "Category" combo box  are not accessible with keyboard.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![categorycombobox and sort by button is not accessible with keyboard](https://user-images.githubusercontent.com/22956130/35725686-19dea3c0-07b7-11e8-98f2-eb76b2cd34c1.png)

**Comment by Eskibear** [comment]
It's the IDE's own dialog, it's out of the toolkit's scope.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749908
